src/gallery/types/artist.ts

```diff
@@ -0,0 +1,4 @@
+export interface Artist {
+  name: string;
+  smartContractNumber: `0x${string}`[];
```

Comment:
Toujours pour info : cette notation 0x, c'est rapport à ça : https://docs.0x.org/introduction/introduction-to-0x ? (pas à la notation en hexadecimal ?)

Reply:
c'est le type demande dans la fonction useContractRead de wagmi